CMakeLists.txt

```diff
@@ -557,5 +558,21 @@ add_subdirectory(ffiTestLibrary ${CMAKE_CURRENT_BINARY_DIR}/build/ffiTestLibrary
 add_third_party_dependencies_per_platform()
 
 # Packaging Setup
-
 include(cmake/packaging.cmake)
+
+# GraphViz Setup
+find_program(DOT dot)
```

Comment:
Good! I'd add a comment explaining what this is doing and why, what do you think?